### ISSUE TITLE
Change the renderer default to UseGPU = false

### ISSUE THREAD

**Issue by pauldendulk** [Mapsui/Mapsui#1470]
There are two bug reports involving problems with the UseGPU = true setting.

https://github.com/Mapsui/Mapsui/issues/1458
https://github.com/Mapsui/Mapsui/issues/1191

[Here](https://github.com/Mapsui/Mapsui/issues/1191#issuecomment-915040958) @charlenni suggested to not change the default because of iOS. I think this issue is too serious to keep it like this. Most developers will not notice it, ship it, and it will only reach the developer if they get proper bug reporting from their customers. 

### Options
- Set it to false (I think this is better than we have right now).
- Set it to false for desktop only. 
- Set is base on runtime information (detect hardware, detect crashes, so a kind of save mode)
- Assist the developer somehow to make the right decision.

**Comment by charlenni** [comment]
This flag decides, if Skia uses the GPU (mostly OpenGL in the background) or only the CPU. If you set the default to CPU, Mapsui will get slow but perhaps more reliable.

Normally a crash means, that the device has problems in the OpenGL implementation as we see in the last issue (update of the graphics driver solvers the problem).

I would stay with the faster version.

**Comment by inforithmics** [comment]
Is there a way to detect if the OpenGpl implementation works? Because in the maui win. Target the gpu needs to be disabled because ski sharp does not yet support gpu acceleration. On Android and iOS the Gpu works.

**Comment by Sebastian1989101** [comment]
@pauldendulk In my apps, I added this option to the settings panel where the user can enable or disable the `UseGPU`. However, the default is `UseGPU = true`, except we detect that the "device" is a emulator, in this case we set it to `false`. So however the default of this is, I set it anyway on each app startup so it won't affect me at all. 

If this option would not be there, I would go for performance as long as it runs stable under iOS (because even tho my apps are the same, I generate ~70% of my income with iOS while having way less work and trouble with it compared to Android, and espacially Google. And also I cannot test on every Android device anyway).
